asreview/entry_points/base.py

```diff
@@ -74,12 +74,15 @@ def _base_parser(prog=None, description=None):
         "--config_file",
         type=str,
         default=None,
-        help="Configuration file with model, balance, and query parameters"
+        help="""Configuration file with model parameters and 
```

Comment:
What is the difference between model parameters and hyperparameters?

Reply:
I think of it as: 

Model parameters are the global settings, like  `balance_strategy = double` 
Hyperparameter values are nested in the model parameters, like `alpha = 1.229374068496514`

```
[global_settings]
n_instances = 1
n_prior_included = 1
n_prior_excluded = 1
model = nb
balance_strategy = double
query_strategy = max
feature_extraction = tfidf

[balance_param]
a = 1.3783791240515941
alpha = 1.229374068496514
b = 0.9153629982662178

[feature_param]
ngram_max = 3
split_ta = 0
use_keywords = 1

[model_param]
alpha = 0.06586028856339346
```

Reply:
I think that might be confusing for the user. Perhaps we can call it simulation settings + model parameters?

Reply:
I'm not a fan of simulation settings as these settings are not confined to simulations. 
How about model settings + model parameters?

Reply:
Sure, that makes sense.